### ISSUE TITLE
Nested folder upload support

### ISSUE THREAD

**Issue by HarrisonB** [filestack/filestack-js#323]
Hi! Is there any plan to enable the picker to select a top-level folder and select all nested folders and their files? I understand this may not be feasible for the `local_file_system` source (cf. #47) but I am curious if this would be possible with other cloud-based sources (e.g. Dropbox).

For example, selecting `Root-Folder` which has a structure like
```
Root-Folder
├── My-Other-File.txt
└── Nested-Folder
    └── My-File.txt
```
would capture both text files.

**Comment by pcholuj** [comment]
Hi @HarrisonB 
For now, we can only do it by using drag and drop, so you can drop the folder into picker and it should add all child files. 
Unfortunately, browsers do not support uploading files by file selection dialog.

**Comment by HarrisonB** [comment]
For completeness (and anyone googling that ends up coming upon this issue): one can use the [`webkitdirectory`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-webkitdirectory) attribute for the `<input>` tag, though this is not on the standard track and suffers from [limited browser support](https://caniuse.com/#feat=input-file-directory). It might be possible for Filestack to make use of this, but I would understand the motivations for staying clear of this API.

**Comment by pcholuj** [comment]
Yes we can use that but it will be only supported in some browsers. We will think about this and i will let you know.

**Comment by shshaw** [comment]
It's been three years. Is file path data still not available via browser APIs that can be utilized by the uploader to preserve folder trees?
